File: 2 Revalida en C++/4 Bucles/11.cpp
///para calcular la planilla de una empresa
#include <bits/stdc++.h>
using namespace std;
int main()
{
    int total=0;
    int contador=0;
    int numero;
    cout<<"ingrese un numero de planilla:";
    cin>>numero;
    while (contador<numero)
	{
	    int horas,salario,pago;
        cout<<"ingrese horas:";cin>>horas;
        cout<<"ingrese salario:";cin>>salario;
        pago=horas*salario;
        total=total+pago;
        contador=contador+1;
        cout<<"el pago es = "<<pago<<endl;
	}
    cout<<"Total es ="<<total;
    return 0;
}

File: 1 Revalida en python/4 Bucles/11.py
#para calcular la planilla de una empresa
total=0
contador=0
numero=int(input("ingrese un numero"))
while contador<numero:
	horas=int(input())
	salario=int(input())
	pago=horas*salario
	total=total+pago
	contador=contador+1
	print pago
print total